playbooks/capsule/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+certs_tar: "/root/{{ ansible_nodename }}.tar.gz"
```

Comment:
Since the vars are global, it payed me off to prefix those with the role name, so that one know later where they come from. It seems I'm not alone with this observation http://probablyfine.co.uk/2014/03/27/how-to-write-an-ansible-role-for-ansible-galaxy/, https://openedx.atlassian.net/wiki/display/OpenOPS/Ansible+Code+Conventions, . Ansible does the same to their vars.